cfme/tests/infrastructure/test_provisioning_dialog.py

```diff
@@ -83,7 +89,7 @@ def _provisioner(template, provisioning_data, delayed=None):
         fill(provisioning_form, provisioning_data, action=provisioning_form.submit_button)
         flash.assert_no_errors()
 
-        request.addfinalizer(lambda: cleanup_vm(vm_name, provider))
+        request.addfinalizer(lambda: remove_vm(vm_name, provider))
```

Comment:
@bargulg  . cleanup_vm uses delete_vm which stops vm if it's not stopped 
https://github.com/ManageIQ/wrapanapi/blob/master/mgmtsystem/rhevm.py#L210 . it works just perfect on VC or SCVMM. have you met any problems else where?